src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -250,6 +256,14 @@ export const SynapseNavDrawer: React.FunctionComponent<
                   iconName: 'delete',
                   onClickGoToUrl: '/#!Trash:0',
                 })}
+              {isInSynapseExperimentalMode() &&
+                (currentUserBundle?.isACTMember ||
+                  currentUserBundle?.isARReviewer) &&
```

Comment:
Notes for isARReviewer state that it is always true for an ACT member, so I don't think you need to ask for USER_BUNDLE_MASK_IS_ACT_MEMBER or reference it here.